lib/Router.js

```diff
@@ -425,26 +427,11 @@ Router.prototype.handle = function(request, connection) {
 
     return Promise.all(callbacks)
     .then(function(results) {
-        results = results.map(function(result) {
```

Comment:
So glad to kill all this unnecessary code that was dealing with lodash's funky merge.